spec/chewy/index/actions_spec.rb

```diff
@@ -54,8 +54,8 @@
 
     context do
       before { DummiesIndex.create }
-      specify { expect { DummiesIndex.create! }.to raise_error }
-      specify { expect { DummiesIndex.create!('2013') }.to raise_error }
+      specify { expect { DummiesIndex.create! }.to raise_error(Elasticsearch::Transport::Transport::Errors::BadRequest) }
```

Comment:
Hmm, I think the errors should be more descriptive. I will add message expectations also.